_posts/2023-12-13-release-0_4_0.md

```diff
@@ -7,7 +7,9 @@ author_url: "https://www.github.com/k-wall"
 categories:  releases kroxylcious topic-encryption
 ---
 
-The Kroxylicious project is very pleased to announce the release of Kroxylicious 0.4.0.   The big news is this release includes our `EnvelopeEncryption` filter which provides an **Encryption-at-Rest** solution for the Apache Kafka(tm) which is _transparent_ to both clients and brokers.
+The Kroxylicious project is very pleased to announce the release of Kroxylicious 0.4.0.
+
+The big news is this release includes our `EnvelopeEncryption` filter which provides an **Encryption-at-Rest** solution for the Apache Kafka(tm) which is _transparent_ to both clients and brokers. We have more work planned to iterate on the implentation to reach production quality. Checkout our [0.5.0 milestone](https://github.com/kroxylicious/kroxylicious/milestones/0.5.0) for details.
```

Comment:
I used EnvelopeEncryption here as that's what we've called the filter - so the argument being consistency.  You could argue that the fact it is using envelope encryption is an implementation detail.  The important thing is what it does - TopicEncryption, or EncryptionAtRest.

Reply:
I think you've expressed it well. We are delivering encryption-at-rest via an envelope encryption filter.